modules/vtt-json.js

```diff
@@ -0,0 +1,53 @@
+function convertVttToJson(vttString) {
+  return new Promise((resolve, reject) => {
```

Comment:
Why is this in the form of a Promise, are you using any asynchronous code?

Reply:
The original library defines the function this way, do you want me to go ahead and make it synchronous?

Reply:
Yeah you could do that, I don't see a need for this to be asynchronous